src/App.tsx

```diff
@@ -1,26 +1,285 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import cn from 'classnames';
 import { UserWarning } from './UserWarning';
+import { useEffect, useRef, useState } from 'react';
+import { TodoItem } from './components/TodoItem/TodoItem';
+import { Todo } from './types/Todo';
+import { TodoStatusFilter } from './types/TodoStatusFilter';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { Notification } from './components/Notifications/Notifications';
+import {
+  todoErrorMessageText,
+  TodoErrorMessage,
+  USER_ID,
+  todoServices,
+} from './api/todos';
+import { CreateTodoForm } from './components/CreateTodoForm/CreateTodoForm';
+import { TodoCreate } from './types/TodoCreate';
 
-const USER_ID = 0;
+const getfilteredTodos = (
+  todos: Todo[],
+  { status }: { status: TodoStatusFilter },
+) => {
+  let filteredTodos = [...todos];
+
+  if (status !== TodoStatusFilter.all) {
+    filteredTodos = filteredTodos.filter(todo => {
+      if (status === TodoStatusFilter.completed) {
+        return todo.completed;
+      }
+
+      return !todo.completed;
+    });
+  }
+
+  return filteredTodos;
+};
```

Comment:
we can simplify the code


```suggestion
const getfilteredTodos = (
  todos: Todo[],
  { status }: { status: TodoStatusFilter },
) => {
  if(status === TodoStatusFilter.all) return todos;
 
  return filteredTodos.filter(todo => {
    if (status === TodoStatusFilter.completed) {
        return todo.completed;
    }
    return !todo.completed;
  });
};
```